usecase/usecase-embedding-android-tests/embeddingapi-async/src/org/xwalk/embedded/api/asyncsample/XWalkViewWithClearCacheForSingleFileAsync.java

```diff
@@ -0,0 +1,209 @@
+// Copyright (c) 2014 Intel Corporation. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package org.xwalk.embedded.api.asyncsample;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.text.DecimalFormat;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.xwalk.core.XWalkInitializer;
+import org.xwalk.core.XWalkResourceClient;
+import org.xwalk.core.XWalkUIClient;
+import org.xwalk.core.XWalkView;
+
+import android.app.Activity;
+import android.app.AlertDialog;
+import android.os.Bundle;
+import android.util.Log;
+import android.view.View;
+import android.view.View.OnClickListener;
+import android.webkit.WebResourceResponse;
+import android.widget.Button;
+import android.widget.TextView;
+
+public class XWalkViewWithClearCacheForSingleFileAsync extends Activity implements XWalkInitializer.XWalkInitListener {
+	private static final String TAG = "XWalkViewWithClearCacheForSingleFileAsync";
+	private static final String EXTRA_CACHE_PATH
+				= "/data/data/org.xwalk.embedded.api.sample/app_xwalkcore/Default/Cache";
```

Comment:
getCacheDir() can only get data/data//cache, this specific path is checked with hengzhi, it is just about xwalk_core as it shows.